src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -221,6 +249,145 @@ public static FamilyType ByName(string name)
             };
         }
 
+        /// <summary>
+        /// Create new Family Instance from Geometry (via SAT import)
```

Comment:
Uhmmmm,  from the method, it looks like that it is to create a FamilyType object. But the comment and the pull request title is saying to create family instance. 

Also if the parameter if geometry, then it is not only via SAT import.